src/app/[locale]/specs/lexicon/en.mdx

```diff
@@ -235,21 +245,31 @@ Unions represent that multiple possible types could be present at this location
 
 By default unions are "open", meaning that future revisions of the schema could add more types to the list of refs (though can not remove types). This means that implementations should be permissive when validating, in case they do not have the most recent version of the Lexicon. The `closed` flag (boolean) can indicate that the set of types is fixed and can not be extended in the future.
 
-A `union` schema definition with no `refs` is allowed and similar to `unknown`, as long as the `closed` flag is false (the default). The main difference is that the data would be required to have the `$type` field. An empty refs list with `closed` set to true is an invalid schema.
+A `union` schema definition with no `refs` is allowed and similar to `unknown`, as long as the `closed` flag is false (the default). The main difference is that the data is required to include the `$type` field.
 
-The schema definitions pointed to by a `union` are objects or types with a clear mapping to an object, like a `record`. All the variants must be represented by a CBOR map (or JSON Object) and must include a `$type` field indicating the variant type. Because the data must be an object, unions can not reference `token` (which would correspond to string data).
+A closed union with only a single type referenced is similar to `ref`. The main difference is that the data is required to include the `$type` field.
+
+An empty closed union (a union with an empty refs list and the `closed` flag set to true) is an invalid schema.
+
+The schema definitions pointed to by a `union` must have type `object` or types with a clear mapping to an object, like a `record`. All the variants must be represented by a CBOR map (or JSON Object) and must include a `$type` field indicating the variant type. Because the data must be an object, unions can not reference `token` (which would correspond to string data).
```

Comment:
I would rejigger both of these paragraphs:

The schema definitions referenced to by a `union` must have type `object` or `record`. All the variants must be represented by a CBOR map (or JSON Object) and must include a `$type` field indicating the variant type.  A `union` can not point to types `query`, `procedure`, `subscription`, `unknown`, `blob`, `cid-link`, `array`, `params`, `ref`, `union`, `token` or any other currently defined types.